api-datatypes.go

```diff
@@ -85,6 +86,12 @@ type UploadInfo struct {
 	ExpirationRuleID string
 }
 
+// RestoreInfo contains information of the restore operation of an archived object
+type RestoreInfo struct {
+	OngoingRestore bool
+	ExpiryTime     time.Time
```

Comment:
It is not obvious what "ExpiryTime" is. Add comments to explain fields.

Do we know the start time of the restore. Would be nice to have, if feasible.

Reply:
> It is not obvious what "ExpiryTime" is. Add comments to explain fields.

Done. Restore API will actually get a copy of the archived object in the first time. The object will be automatically removed from the fast tier after x days.

> Do we know the start time of the restore. Would be nice to have, if feasible.

No, S3 doesn't expose that.